### PR TITLE
Adds #286 - /loginmsg set <message>

### PR DESCRIPTION
**Pull request by DUFCLiam**
#286

### CODE REVIEW

**Comment by AvalancheYT** [comment]
gg, same code as mine but I found some mistakes. There's nothing defined in delete argument, and also add player support?

**Comment by DUFCLiam** [comment]
I have corrected some things.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_loginmsg.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        
+        if (!args[0].equalsIgnoreCase(commandSender) && !plugin.al.isSeniorAdmin(sender))
+        {
+            msg("You don't have permission to assign another users login message!");
+            return true;
+        }
+        
+        if (args[1].equalsIgnoreCase("delete"))
+        {
+            admin.setLoginMssgae(null);
```
Comment (rovertdude) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_loginmsg.java]: seems to have some typos like right here I believe.

**Comment by RobinGall2910** [comment]
...you should cut down on the commits...

**Comment by DUFCLiam** [comment]
@RobinGall2910, I am trying but there is a new GitHub feature: "Squash commits and merge" then you can just merge all the commits as one; Also Jerom does not merge them. He adds them himself.

**Comment by Wild1145** [comment]
@RobinGall2910 @DUFCLiam That is correct, the squash commits just merges everything into a single commit and then pushes the commit, its incredibly useful.

**Comment by DUFCLiam** [comment]
@Wild1145, I know! I noticed this on my own one; I disabled it at first now I only allow Squash commits.

**Comment by OxLemonxO** [comment]
Don't need this anymore. Please close this @DUFCLiam

**Comment by DUFCLiam** [comment]
You could state a reason why but I already know why dues to the implementation of `/myadmin`

**Pull request closed without merging**